### DISCUSSION TITLE
CFWheels files that should be excluded from commit

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1318] [Q&A] [answered]
CFWheels newbie here, what files are recommended to be excluded from a app project that goes through development, test, and productions servers? I use Commandbox to dev in but our test and production do not use it.

I will guess the following list, but please advise/modify.

CFConfig.json (or wherever the CF Config output was written to)
box.json
server.json

Perhaps also:

readme.md?

**Comment by bpamiri** [comment, marked as answer]
Our environment is similar in that all our developers use a CommandBox environment to develop, we use git and GitHub for source control and Jenkins for CI/CD. Code committed to the Develop branch automatically gets pushed to Staging, and commits to the Master branch automatically get pushed to the Productions servers.

The files you mentioned, are all checked into source control because all the developers need them to insure their development environments are the same. These files get pushed to staging and productions even though they are not needed or used in those environments. Lucee will ignore them since those environments are hand crafted.

**Reply by cfcoder** [reply]
We don't use Lucee but ACF 2018 and subversion. We are old school and file movement is not automated usually. As far as I know all three json files mentioned are used in the Commandbox environment. Outside of that it's ignored.

**Reply by bpamiri** [reply]
That is correct. They are not needed by the framework outside of a CommanBox or development environment.
